### ISSUE TITLE
Add editor panel undo button

### ISSUE THREAD

**Issue by barnettwilliam** [mdenet/educationplatform#141]
There is no undo button for editor panels. Feedback from the MDENet Symposium indicated that an undo button would be helpuful so that any changes made can be easily reverted and would reassure learners who are unfamiliar with the models/languages used when modifying the panel contents.

**Comment by jgsuess** [comment]
This requires command pattern and sessions, I am unsure if these are part of the implementation. You would need a framework that does this from the start.

**Comment by agarciadom** [comment]
This may just be part of the Ace editor or even the browser's basic functionality. If I just go to the [Ace website](https://ace.c9.io/) and type something into their demo editor, I can then press Ctrl+Z for undo. I assume this functionality will be available through some browser/Ace editor API.

**Comment by szschaler** [comment]
Yes, ACE has an `UndoManager` that will make this trivial to add. It's just about making available existing functionality also through a button.
